### DISCUSSION TITLE
Confused about statuscolumn

### DISCUSSION THREAD

**Discussion by Jomik** [LazyVim/LazyVim#1817] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/699655/a8c0332c-fdf9-4205-ad3b-955bf271ffec)
What does G and D, etc. mean in status column?

**Comment by dpetka2001** [comment, marked as answer]
I believe they are `marks`.

**Reply by dpetka2001** [reply]
If you go to another line and then do `'G` it should get you to that line where `G` is set.

**Reply by Jomik** [reply]
Thanks, yeah of  course.. Woops.

**Comment by folke** [comment]
Global marks indeed. You probably set them by accident? 😅

**Reply by Jomik** [reply]
Daaaayum. I am not used to them being displayed. Haha 😅
